schema/changelog-3.8.xml

```diff
@@ -8,6 +8,24 @@
 
   <changeSet author="author" id="changelog-3.8">
 
+    <createTable tableName="device_aliases">
```

Comment:
Yes we did https://github.com/tananaev/traccar/issues/2302#issuecomment-246158247
Anyway, `attribute_aliases` is much clearer